core/req-view-change.go

```diff
@@ -83,11 +83,11 @@ func makeReqViewChangeProcessor(collect reqViewChangeCollector, startViewChange
 	}
 }
 
-func makeReqViewChangeCollector(f uint32) reqViewChangeCollector {
+func makeReqViewChangeCollector(commitCertSize, n uint32) reqViewChangeCollector {
 	var (
 		view      uint64
-		collected = make(messages.ViewChangeCert, 0, f+1)
-		replicas  = make(map[uint32]bool, f+1)
+		collected = make(messages.ViewChangeCert, 0, n-commitCertSize+1)
```

Comment:
In my first look, I though this change is beyond the simple cleanup, but as you [mentioned](https://github.com/hyperledger-labs/minbft/discussions/215#discussioncomment-1822594) in #215 more than `f+1` ReqViewChange message can be required for generic case of `n > 2f + 1`. So this change is justified as such.

Reply:
Yes, the old code assumed `n=2f+1`.